pkg/gds/admin.go

```diff
@@ -794,6 +794,7 @@ func (s *Admin) ListVASPs(c *gin.Context) {
 			// Add certificate serial number if it exists
 			if vasp.IdentityCertificate != nil {
 				snippet.CertificateSerial = fmt.Sprintf("%X", vasp.IdentityCertificate.SerialNumber)
+				snippet.CertificateExpiration = vasp.IdentityCertificate.NotAfter
```

Comment:
I don't remember what we're checking in the tests, is it easy to add a check in the test to verify that we are populating this value?

Reply:
Unfortunately it's not easy - we don't have certificate data in our fixtures to test this (or at least in the small fixtures).